commands/outliers.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+const co = require('co')
```

Comment:
For what it's worth, pretty sure we can use `async/await` directly in the CLI instead of `co`, but I'm okay leaving this here for consistency (especially since it's a port of an existing command), and then porting all the commands to `async/await` in bulk.